src/commands/analyze/command.ts

```diff
@@ -637,6 +710,71 @@ export async function analyzeCommand(options: GenezioAnalyzeOptions) {
     log.info(result);
 }
 
+async function analyzeEnvironmentFilesConcurrently(
+    envExampleFiles: Map<string, string>,
+    genezioConfig: RawYamlProjectConfiguration,
+): Promise<
+    Map<
+        string,
+        {
+            environmentVariables: ProjectEnvironment[];
+        }
```

Comment:
The return type `Map<string, { environmentVariables: ProjectEnvironment[] }> ` is repeated multiple times. Define it as a separate type for maintainability:

```
type EnvironmentAnalysisResult = Map<string, { environmentVariables: ProjectEnvironment[] }>;
```